query-engine/connectors/sql-query-connector/src/query_builder/read/many_related_records/base_query.rs

```diff
@@ -29,9 +28,11 @@ impl<'a> ManyRelatedRecordsBaseQuery<'a> {
 
         let condition = query_arguments
             .filter
+            .clone()
             .map(|f| f.aliased_cond(None))
             .unwrap_or(ConditionTree::NoCondition);
 
+        let order_directions = query_arguments.ordering_directions();
```

Comment:
not sure without reading the whole context, but maybe the could be removed if you put this line before the `let condition = ...`. It's not super important but since it's the query engine we have to avoid cloning if we can.

Reply:
Yes, that worked.